swagger.yaml

```diff
@@ -3349,7 +3386,7 @@ parameters:
     in: query
     description: |
       Direction of the sorting. The default direction is `desc`. Available
-      parameters are: `asc` and `desc`
+      parameters are: `asc`, `desc` and 'rand'. Parameter `rand' returns randomly sorted elements
```

Comment:
is this true, everywhere this snipped is used?

Reply:
Yes, it should work everywhere after the change of lib/bl/core/logic/sort_collection.rb here https://github.com/brandslisten/bl-core-logic/pull/329/files